huntsman/tests/test_camera.py

```diff
@@ -0,0 +1,347 @@
+import pytest
```

Comment:
Do you want to just name this `test_pyro_camera.py` and then we can leave it for only Pyro items (if we ever add other camera types just to Huntsman)?

Reply:
Inclined to leave as is to emphasise that this is a lightly edited copy of `pocs/tetsts/test_camera.py`, and it supposed to be kept in sync with that.